Simulation.sln

```diff
@@ -69,12 +69,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{BC562DAE-FE2
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Simulation", "Simulation", "{9008B252-2DF7-404B-B626-D4497BB70A05}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Quantum.QSharp.Foundation", "src\Simulation\QsharpFoundation\Microsoft.Quantum.QSharp.Foundation.csproj", "{DB45AD73-4D91-43F3-85CC-C63614A96FB0}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Quantum.QSharp.Foundation", "src\Simulation\QsharpFoundation\Microsoft.Quantum.Qsharp.Foundation.csproj", "{DB45AD73-4D91-43F3-85CC-C63614A96FB0}"
```

Comment:
For comparison, it seems that C# and F# are both consistently written with a capital S (CSharp and FSharp).

Reply:
> a lot less invasive than changing all Sdk properties

[MSBuild property names are case-insensitive.](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild?view=vs-2019#BKMK_Items)

Reply:
While MSBuild doesn't care, Linux and dotnet core do, which is why this worked fine for local build and failed e2e. If there is a casing difference, Linux's case-sensitive file system will say the project reference is not found, and fail compilation with missing dependencies.

I agree with Bettina that we should determine a consistent pattern. For now I'm just going to try and avoid casing changes for existing files and ensure my new files compile, and then we can revisit this its own effort. I filed #449 to track this.

Reply:
Yes, I didn't mean that we shouldn't use consistent casing - I meant that I don't think changing the MSBuild properties to use capital S (currently they're written Qsharp) would be an invasive change, since it seems like MSBuild ignores case differences.

Reply:
Ah, I see what you mean. Yes, that makes sense!